package.nuspec

```diff
@@ -9,33 +9,16 @@
     <repository type="git" url="$repoUri$" commit="$commitId$"/>
   </metadata>
   <files>
-    <!-- iOS device debug -->
-    <file src="DerivedData\Build\Products\Debug-iphoneos\FRNAvatar\libFRNAvatar.a" target="Debug-iphoneos"/>
```

Comment:
with empty debug target that is still ok? how would we ever debug native layers of RN?

Reply:
We never had a full end to end debug story working for the FURN static libraries in devmain. If we wanted to, we would need to write a custom script to build the libraries locally, then side load them into devmain. That script could just build debug libraries with dsyms, and pack them into a "ship-$(PLATFORM_NAME)" directory inside the NuGet and devmain would be none the wiser when we side load that NuGet. 

In summary, I don't think this change would prevent us from debugging FURN native code in devmain in the future, we still need to side load with a custom script.